include/dlaf/factorization/cholesky/mc/cholesky_L.h

```diff
@@ -144,10 +147,13 @@ void cholesky_L(comm::CommunicatorGrid grid, Matrix<T, Device::CPU>& mat_a) {
         // Avoid useless communication if one-column communicator and if on the last column
         if (col_comm_size > 1 && k != (mat_a.nrTiles().cols() - 1)) {
           // Broadcast the panel column-wise
-          hpx::dataflow(hpx::util::unwrapping([](auto&& tile, auto&& comm_wrapper) {
-                          comm::sync::broadcast::send(comm_wrapper().colCommunicator(), tile);
-                        }),
-                        mat_a.read(kk), serial_comm());
+          auto send_bcast_f =
+              hpx::util::unwrapping([ex = executor_mpi_col](auto&& tile, auto&& promise) mutable {
+                auto bcast_fut = comm::bcast(ex, ex.comm().rank(), tile);
+                promise.set_value();
+                bcast_fut.get();
+              });
+          hpx::dataflow(std::move(send_bcast_f), mat_a.read(kk), col_bcast.chain());
```

Comment:
communication tasks should be high priority, or even better be synchronous (i.e. executed as soon as all the dependencies get ready). I think that by default dataflow creates a normal priority async task.

Reply:
send/recv tasks now use `executor_hp`.